### PR TITLE
Python 3 travis

### PR DESCRIPTION
**Pull request by disconnect3d**
I'm not sure if I can push local changes to https://github.com/trailofbits/manticore-examples/pull/22, so I am sending this PR to push that PR to be merged (and closing linked PR).

TLDR: This PR is going to make travis work.

### CODE REVIEW

File: ais3_crackme/win.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def hook(state):
     state.constrain(solution[4] == ord('{'))
     buffer_addr=state.cpu.read_int(state.cpu.RAX)
     m.context[1] = buffer_addr
-    print ("buffer addr : %08x " %(buffer_addr))
+    print ("buffer addr : {:08x}".format(buffer_addr))
```
Comment (disconnect3d) [inline ais3_crackme/win.py]:
```suggestion
    print(f"buffer addr : {buffer_addr:08x}")
```

File: ais3_crackme/win.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def hook(state):
     print("it is a win path")
     buffer_addr = m.context[1]
     res = ''.join(map(chr, state.solve_buffer(buffer_addr,num_bytes)))
-    print("flag is : %s"%(res))
+    print("flag is : {}".format(res))
```
Comment (disconnect3d) [inline ais3_crackme/win.py]:
```suggestion
    print(f"flag is : {res}")
```

File: exploit_generation_example/crash_analysis.py [hunk 3]
```diff
@@ -29,11 +29,11 @@ def record_trace(state):
     # Print number of instructions recorded
 
     with m.locked_context() as c:
-        print "%d instructions are recorded" % len(c['trace'])
+        print("{} instructions are recorded".format(len(c['trace'])))
```
Comment (disconnect3d) [inline exploit_generation_example/crash_analysis.py]:
```suggestion
        print("{len(c['trace'])} instructions are recorded")
```

File: exploit_generation_example/record.py [hunk 4]
```diff
@@ -18,10 +18,11 @@ def record_trace(state):
         c['trace'] += [pc]
 
         # We manipulate directly capstone instruction
-        c['last_ins'] = "%s %s" % (ins.mnemonic, ins.op_str)
+        c['last_ins'] = "{:s} {:s}".format(ins.mnemonic, ins.op_str)
```
Comment (disconnect3d) [inline exploit_generation_example/record.py]:
```suggestion
        c['last_ins'] = f"{ins.mnemonic} {ins.op_str}"
```

File: exploit_generation_example/crash_analysis.py [hunk 5]
```diff
@@ -64,7 +64,7 @@ def follow_trace(state):
 
         # We stop to explore the current path if it doesn't follow the targeted path
         if not state.context['visited'] <= trace_set:
-            print "State diverge at 0x%x" % state.cpu.PC
+            print("State diverge at 0x{:x}".format(state.cpu.PC))
```
Comment (disconnect3d) [inline exploit_generation_example/crash_analysis.py]:
```suggestion
            print(f"State diverge at 0x{state.cpu.PC:x}")
```

**Comment by ehennenfent** [comment]
Closing in favor of #37

**Pull request closed without merging**